engines/director/lingo/lingo-the.cpp

```diff
@@ -1730,6 +1730,16 @@ void Lingo::setTheSprite(Datum &id1, int field, Datum &d) {
 					channel->_dirty = true;
 				}
 				channel->setCast(castId);
+
+				// If we change the channel cast we also have to update the sprite in the score.
```

Comment:
This requires a test movie with the original

Reply:
Please join our Discord, ping me, and I will explain and give you all the needed tools